packages/styles/src/components/progress.scss

```diff
@@ -10,7 +10,7 @@
 @use '../mixins/utilities';
 
 .progress-bar {
-  color: var(--contrast-color, progress-bars.$progress-bar-color);
+  color: var(--post-contrast-color, progress-bars.$progress-bar-color);
```

Comment:
The fallback `progress-bars.$progress-bar-color` should not be needed here. What do you thing?

Reply:
Statement is entirely unnecessary. Removed it.